lib/utils/resolve-version.js

```diff
@@ -53,7 +53,8 @@ module.exports = function resolveVersion(version, update) {
         } catch (e) {
             return Promise.reject(new errors.CliError({
                 message: 'Ghost-CLI was unable to load versions from Yarn.',
-                log: false
+                log: false,
+                err: e
```

Comment:
`log` is false. So we don't log into file.

And we only log the original error in verbose mode. That means I won't see the original error?